_includes/nav.html

```diff
@@ -1,7 +1,7 @@
-<nav class="navbar is-info is-fixed-top" role="navigation" aria-label="main navigation">
+<nav class="navbar is-fixed-top is-link" role="navigation" aria-label="main navigation">
   <div class="container" id="container">
     <div class="navbar-brand">
-        <img class="navlogo" style="width:50px;height:52px;" src="/img/favicon.png">
```

Comment:
Was there a reason for this? It causes the bug found at #9 and also affects Safari 12.

Reply:
Oh I got it

![image](https://github.com/TheBobPony/bobpony.com/assets/80969068/94738c9e-4513-4011-81ad-298c4440ed56)

Reply:
@Smu1zel Maybe can you open that PR again? :)

Reply:
> @Smu1zel Maybe can you open that PR again? :)

Sure

Reply:
It's now merged. 

Regardless, thank you for the PR.